shell.nix

```diff
@@ -36,11 +39,9 @@ in haskell.packages.shellFor {
     dev.packages.purty
     dev.packages.purs
     dev.packages.spago
-    pkgs.stack
     dev.scripts.fixStylishHaskell
     dev.scripts.fixPurty
     dev.scripts.updateClientDeps
-    pkgs.python3 # for @reactormonk who's running a python web server
```

Comment:
:-(

Reply:
It's already in there for Sphinx, so this was redundant :)